client/verta/verta/_registry/modelversion.py

```diff
@@ -752,11 +756,16 @@ def add_attributes(self, attrs):
 
         # build KeyValues
         attribute_keyvals = []
+        if overwrite:
+            existing_attrs = {}
+        else:
+            existing_attrs = self.get_attributes()
```

Comment:
Because you have `if not key in existing_attrs or overwrite:` later, this `if overwrite` I think is unnecessary, right?
```suggestion
        existing_attrs = self.get_attributes()
```

Reply:
Yes, it is unnecessary. It is used just to not call get_attributes if overwrite=True.